visualization/app/codeCharta/ui/codeMap/rendering/codeMapVertexShader.glsl

```diff
@@ -41,11 +42,27 @@ void getDirectionalDirectLightIrradiance(const in DirectionalLight directionalLi
 
 void main() 
 {
-    vec4 worldPosition = modelMatrix * vec4(position, 1.0);
+    vec3 adjustedHeightPos = position;
+    // height position will be adjusted based on camera position in order to prevent 
+    // z-fighting even in far away zoom levels when position is positive.
+    // preferred multiplication over if branching for the isHeight variable 
+    // the single line could be written this way : 
+    //
+    // adjustedHeightPos = position
```

Comment:
I would just use the more extensive if-statement instead of the one-liner. So we can remove the comment and don't have to adjust the comment and the code itself on every change.

Reply:
Sure, I just don't like branching branching in the GPU side as it might lead to more computation. There should be no performance issue in this case. I will change it and remove the comments.

Reply:
Ah ok, I didn't know that. :)

Reply:
should I leave it as it is or change it ? It's up to you :)

Reply:
Just as you like. ;) I am fine with both solutions.